src/multiselectMixin.js

```diff
@@ -318,9 +320,6 @@ module.exports = {
       this.adjustSearch()
     },
     'search' () {
-      /* istanbul ignore else */
-      if (this.search === this.currentOptionLabel) return
```

Comment:
That’s fine – I’ve made some changes to the way `this.search` behaves that also made this line obsolete.